homework-1/task-3/main.js

```diff
@@ -0,0 +1,92 @@
+const bank = document.querySelector('.bank');
+const totalMoney = document.querySelector('.total');
+const moneyInput = document.querySelector('.number-money');
+const moneyBtn = document.querySelector('.money-btn');
+const render = document.querySelector('.render')
+
+let quantityMoney = [];
+
+let limits = {
+    1000: 5,
+    500: 5,
+    100: 5,
+    50: 34,
+    20: 10,
+    10: 10,
+};
+
+
+let quantity = Object.entries(limits);
+
+quantity.forEach((value) => {
+   sum = +value[0] * value[1];
+   quantityMoney.push(sum);
+})
+
+let moneyTotal = quantityMoney.reduce((acc, value) => {
+    return acc + value;
+}, 0)
+
+for (const key in limits) {
+    let sum = +key * limits[key];
+    let p = document.createElement('p');
+    p.innerHTML = `В банкоматі такі купюри ${key} в кількості ${limits[key]} = ${sum}`;
+    bank.append(p);
+}
+
+totalMoney.innerHTML = `Загальна сума грошей ${moneyTotal - moneyInput.value}`;
+
+
+
+let iWantToGet = (ammountRequired, limits) => {
```

Comment:
Where you have found this solution?

Reply:
in internet...but the show in page, and counting, change in the amount of currency total amount is i did myself